include/yaclib/util/intrusive_ptr.hpp

```diff
@@ -18,6 +18,8 @@ class IntrusivePtr final {
   static_assert(std::is_base_of_v<IRef, T>, "T must be derived class of IRef");
 
  public:
+  using Pointee = T;
```

Comment:
```suggestion
  using Value = T;
```

https://en.cppreference.com/w/cpp/iterator/iterator_traits.html